spec.html

```diff
@@ -15692,21 +15791,73 @@ <h1>Runtime Semantics: BindingInitialization</h1>
         <emu-alg>
           1. Return NormalCompletion(~empty~).
         </emu-alg>
+        <emu-grammar>
+          ObjectBindingPattern :
+            `{` BindingPropertyList `}`
+            `{` BindingPropertyList `,` `}`
+        </emu-grammar>
+        <emu-alg>
+          1. Perform ? PropertyBindingInitialization for |BindingPropertyList| using _value_ and _environment_ as the argument.
+          1. Return NormalCompletion(~empty~).
+        </emu-alg>
+
+        <emu-grammar>ObjectBindingPattern : `{` BindingRestProperty `}`</emu-grammar>
+        <emu-alg>
+          1. Let _excludedNames_ be a new empty List.
+          1. Return the result of performing RestBindingInitialization of _BindingRestProperty_ with _value_, _environment_ and _excludedNames_ as the arguments.
+        </emu-alg>
+
+        <emu-grammar>ObjectBindingPattern : `{` BindingPropertyList `,` BindingRestProperty `}`</emu-grammar>
+        <emu-alg>
+          1. Let _excludedNames_ be the result of performing PropertyBindingInitialization of |BindingPropertyList| using _value_ and _environment_ as arguments.
+          1. ReturnIfAbrupt(_excludedNames_).
+          1. Return the result of performing RestBindingInitialization of _BindingRestProperty_ with _value_, _environment_ and _excludedNames_ as the arguments.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="Rest-RuntimeSemantics-PropertyBindingInitialization">
```

Comment:
As for destructuring assignment evaluation, I'd keep these semantics in the current `<emu-clause>`.

Reply:
Could you clarify here please?

Reply:
I think he's suggesting that you can just add the new production under the existing clause rather than adding a new clause for it.